src/components/RangeSlider/components/SingleThumb/SingleThumb.scss

```diff
@@ -33,6 +33,22 @@ $range-thumb-shadow-focus: 0 0 0 rem(1px) color('indigo');
     background-color: transparent;
     cursor: pointer;
   }
+
+  &::after {
+    content: '';
+    position: absolute;
+    height: $range-track-height;
+    width: 100%;
+    background-image: linear-gradient(
+      to right,
+      var(--p-surface-inverse-foreground),
+      var(--p-surface-inverse-foreground) 50%,
+      transparent 50%,
+      transparent 100%
+    );
+    background-size: $range-track-height $range-track-height;
```

Comment:
We tried omitting the second parameter (the height) and letting it default to `auto`, though this doesn't seem fully supported on Safari.